src/renderer/icon-themes/icon-pickers/base64-picker.ts

```diff
@@ -11,25 +11,31 @@
 import { IIconPicker } from '../icon-theme-registry';
 
 /**
- * An icon picker which is only a text area. The user can enter a base64 encoded image
- * directly.
+ * An icon picker which is only a text area. The user can enter a base64 encoded image or
+ * URL directly.
  */
 export class Base64Picker implements IIconPicker {
   /** The container to which the icon picker is appended. */
   private fragment: DocumentFragment = null;
 
-  /** The textarea where the user can enter a base64 encoded image. */
+  /** The textarea where the user can enter a base64 encoded image or URL. */
   private textArea: HTMLTextAreaElement = null;
 
-  /** This callback is called when the user enters a valid base64 encoded image. */
+  /** This callback is called when the user enters a valid base64 encoded image or URL. */
   private onSelectCallback: (icon: string) => void = null;
 
   /** Creates a new IconPicker and appends it to the given container. */
   constructor() {
     this.fragment = document.createDocumentFragment();
 
     this.textArea = document.createElement('textarea');
-    this.textArea.placeholder = 'data:image/svg;base64,...';
+    this.textArea.placeholder = [
+      'Base64 example:',
+      'data:image/svg;base64,...',
+      '',
+      'URL example:',
+      'https://cdn.simpleicons.org/simpleicons/white',
+    ].join('\n');
```

Comment:
This would need to be translatable (using this `i18next.t(...)` syntax you will find in other places in the code). Or you come up with something which does not need a translation....

Once you added one of the `i18next.t` calls, you can run `npm run i18n` to update the translation template in `locales/en/translation.json`.